### ISSUE TITLE
OGCS fails to start after Office upgrade [0x8002802B]

### ISSUE THREAD

**Issue by sanojn** [phw198/OutlookGoogleCalendarSync#1720]
**OGCS Version**: v2.10.1.0-alpha  
**Edition**: Installed
**Sync Direction**:n/a
Outlook version: 2309 (Build 16827.20166)

**Bug Description**  
An error message pops up when starting OGCS.
See attached screen shot.

**Steps to Reproduce the Issue**  
1. Crash appears when autostarted after login or by start OGCS by clicking on desktop shortcut.

Repairing the Outlook installation didn't help.

![Clipboard01](https://github.com/phw198/OutlookGoogleCalendarSync/assets/58303304/892ff999-05b8-4138-b7d3-5fdbbd149587)
[OGcalsync.log.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/12965456/OGcalsync.log.zip)

**Comment by phw198** [comment]
Looks like the same issue as #1710 which has a hotfix - that should at least redirect you to the wiki, if you could confirm that works?

**Comment by sanojn** [comment]
The hotfix does redirect to the Wiki, though it doesn't provide a solution.
It points me to issue #999 which seems old. The solutions provided there don't help.

I can confirm that the error started to appear after an Office upgrade.
Repairing doesn't help. I can't run the dependencies executable that is mentioned in #999.

One comment in #999 states that the problem is a registry entry concerning a specific DLL (Ole32.dll), but there is no information on how to find that entry or what a correct entry would look like. Do you have any hints?

**Comment by phw198** [comment]
@sanojn You're referring to this comment?

> This problem occurs if the Microsoft Office part (Ole32.dll file that is located in the %WINDIR%\System32 folder) is not registered correctly in the Microsoft Windows registry.

Instructions for that are also covered in [this section of the wiki](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/FAQs---COM-Errors#0x80040155---interface-not-registered) - let me know if that helps.

**Comment by sanojn** [comment]
Hi Paul,

Yes, that was what I was referring to. Not sure that it’s supposed to apply to
my error. It seems that some commenters are confusing different error codes
so I’m not sure what fixes actually applies to my error code (ending in 2B).

Nevertheless, I went through all steps I could find in the wiki that relates to the registry.

Everything sems fine. The only discrepancy was the upper/lower-casing of
…/root/… vs …/Root/… in the path to MSOUTL.OLB.
I fixed that, but it didn’t help.

I may have to uninstall and reinstall office completely.

Jonas
